src/components/twilio-voice-dialer/public/index.html

```diff
@@ -9,7 +9,9 @@
   <body>
     <h1>Twilio Voice Dialer</h1>
 
-    <twilio-voice-dialer recipient register></twilio-voice-dialer>
+    <twilio-voice-dialer recipient register>
+      <twilio-voice-basic-call-control></twilio-voice-basic-call-control>
```

Comment:
We don't want this reference to have the hold and resume functionality right? This is the voice dialer without any conference/call control stuff?

Reply:
good catch. must have missed this in my rebase